router/router.gno

```diff
@@ -217,11 +247,17 @@ func finalizeSwap(inputToken, outputToken string, resultAmountIn, resultAmountOu
 
 	if swapType == "EXACT_IN" {
 		if !tokenAmountLimit.Lte(afterFee) {
-			panic(ufmt.Sprintf("[ROUTER] router.gno__finalizeSwap() || Too few received for user (expected minimum: %s, actual: %s, swapType: %s)", tokenAmountLimit.ToString(), afterFee.ToString(), swapType))
+			panic(addDetailToError(
+				errInvalidInput,
+				ufmt.Sprintf("router.gno__finalizeSwap() || Too few received for user (expected minimum: %s, actual: %s, swapType: %s)", tokenAmountLimit.ToString(), afterFee.ToString(), swapType),
+			))
 		}
 	} else {
 		if !resultAmountIn.Lte(tokenAmountLimit) {
-			panic(ufmt.Sprintf("[ROUTER] router.gno__finalizeSwap() || Too much spent for user (expected maximum: %s, actual: %s, swapType: %s)", tokenAmountLimit.ToString(), resultAmountIn.ToString(), swapType))
+			panic(addDetailToError(
+				errInvalidInput,
+				ufmt.Sprintf("router.gno__finalizeSwap() || Too much spent for user (expected maximum: %s, actual: %s, swapType: %s)", tokenAmountLimit.ToString(), resultAmountIn.ToString(), swapType),
+			))
```

Comment:
This does not appear to be an error in the input.
It appears to be an error out of slippage.

Reply:
f146719bed956e83b479873fd65361de0fdefaba